### ISSUE TITLE
Hover State has inappropriate text on screen

### ISSUE THREAD

**Issue by satkanhub** [topcoder-platform/community-app#2567]
**Describe the bug**
Hover State has inappropriate text on screen

**To Reproduce**
1. Open the test URL
2. Reduce the window size of brwoser until you see Menu bar at the top left.
3. Hover on Menu Bar and observe

**Expected behavior**
Hovering on the menu bar at the left should not display any inappropriate text.

**Screenshots**
![image](https://user-images.githubusercontent.com/51492004/59144107-e8992980-89d2-11e9-8767-737249fdfc80.png)

**Desktop:** 
 - OS: Windows 10 Home
 - Browser : Chrome 
 - Version :  75.0.3770.80

**Comment by nithyaasworld** [comment]
similar to https://github.com/topcoder-platform/community-app/issues/2559

**Comment by satkanhub** [comment]
@nithyaasworld Even though the issue is similar, is this still not a new bug ? I understand duplicate, but what it means to **Similar** which i am unable to understand. Untill we find the issues on all objects. it was never know on where the similar issues are ?

**Comment by nithyaasworld** [comment]
@satkanhub Basically all elements on the navigation component seem to have a tooltip with some incorrect value. 

So whenever you see some issues that is common to all elements, you should be raising it as a single bug. Not a separate bug. You can add your findings as a comment to your first-bug. This is a general practice we should follow in bug hunt. 

This has also been stated in the Challenge Details Page.

I have a general advise for you! Please make sure you read the Challenge Details and all the documents provided in the challenge Test Phase Document Forum thoroughly, before starting to work on it. So you will know what is expected and can avoid raising Out of Scope bugs.

Please post any questions you have regarding this Bug Hunt in the challenge forum. I might miss your comment in the hundreds of Github-mails I receive.
